service/src/main/java/bio/terra/cbas/dependencies/leonardo/AppUtils.java

```diff
@@ -33,26 +34,41 @@ public AppUtils(
   }
 
   int appComparisonFunction(ListAppResponse a, ListAppResponse b) {
```

Comment:
is this function still being used?

Reply:
Currently no, I left it thinking that it may be able to be reused in a following ticket. It can be removed for clarity, however.